### PR TITLE
Allow filtering/searching by account ID

### LINKED ISSUE #69: Allow filtering/searching by account ID
Thanks for implementing a search box. It would be great if this also supported searching by account ID.

### PR DESCRIPTION
**Pull request by heldersepu**
Adding a new option to allow searching on the account ID.
By default the option is disabled, not to change the current behavior

Fixes #69

### CODE REVIEW

File: test/content_scripts.test.js [hunk 1]
```diff
@@ -323,5 +323,63 @@ describe('ContentScripts', () => {
...
+        extendIAMFormList();
+        
+        const filter = document.querySelector('#AESR_RoleFilter')      
+        filter.value = "666611115555"
+        filter.dispatchEvent(new KeyboardEvent('keyup',{'key':'5'}))
+
+        const roles = Array.from(document.querySelectorAll('#awsc-username-menu-recent-roles li[style*="display: block;"]'))
+        expect(roles.length).to.eq(1);
+        expect(roles[0].querySelector('input[name="roleName"]').value).to.eq('renpou');        
+      })
```
Comment (tilfin) [inline test/content_scripts.test.js]: Good! 🥇

**Comment by tilfin** [comment]
@heldersepu Thank you so much. I was going to change the default behavior, not to implement the option to switch including account ID. Do you feel the need to add the option?

**Comment by heldersepu** [comment]
> @heldersepu Thank you so much. I was going to change the default behavior, not to implement the option to switch including account ID. Do you feel the need to add the option?

@tilfin When I contribute for the first time to project I prefer not to make force my changes, but make them optional...

I think the option is a safe bet not to disturb current behavior that most users are used to, I would release it like this and in the future if users ask to enable it by default we can easy do that

File: test/content_scripts.test.js (hunk 1 above)
Comment (heldersepu) [reply test/content_scripts.test.js]: Yes, this increases code coverage on the project 👍

**Comment by tilfin** [comment]
@heldersepu OK, but this extension has already supported `Hide account id` so the value of `<input name="displayName">` does not contain the Account ID if it is true. To add an option makes the specfication complex so I hope to change the default behavior because the change does not disable to find by a profile name.

**Comment by heldersepu** [comment]
@tilfin thanks for pointing out that `Hide account id` I never use that option before and did not consider it while I was coding this PR, will refactor it to keep it in mind

**Comment by tilfin** [comment]
@heldersepu Please do not add the `includeAccountIdInSearch` option. It is easier to understand the feature if you can search the only displayed information. I think that the users who sets `hidesAccountId` true need not to be able to find profiles by hidden Account ID. I do not want to add an option as much as possible because adding a option makes us maintain this extension more difficult.

**Comment by heldersepu** [comment]
@tilfin I removed the `includeAccountIdInSearch` option.
Now by default it will search on the Account ID and those users that set `hidesAccountId` true will not be able filter on it Account ID.

**Comment by tilfin** [comment]
@heldersepu Thank you for accepting my request.

**Comment by heldersepu** [comment]
@tilfin Is there anything else we need to merge this PR?

**Comment by heldersepu** [comment]
@tilfin when can we expect a new release?

**Pull request merged**